docs/source/get-started.mdx

```diff
@@ -46,18 +49,18 @@ kotlin {
 
 ## Execute your query
 
-Create an `ApolloClient`:
+In your application code, create an instance of `ApolloClient`, providing your GraphQL server's URL where indicated:
 
 ```kotlin
-// First, create an `ApolloClient`
 // Replace the serverUrl with your GraphQL endpoint
 val apolloClient = ApolloClient(serverUrl = "https://your.domain/graphql")
 ```
 
-Apollo Android uses coroutines to handle concurrency. If you're using Java, see [RxJava2](usage/rxjava2) for an alternative way to execute your queries.
+Now you can execute your defined query with `ApolloClient` like so:
 
-```
-// Replace `runBlocking {}` by `viewModelScope.launch {}` or any other coroutine scope builder
+```kotlin
+// You can replace `runBlocking {}` with `viewModelScope.launch {}`
```

Comment:
Out of curiosity, what are the reasons to choose `"You can replace"` vs the more imperative `"Replace"`? I never know which one to use.

Reply:
This actually depends on whether I am correct with my edit, so let me know if I'm incorrect!

My edit assumes that `runBlocking` is a valid option here, meaning the reader doesn't _have_ to replace it for this code to work. Because this is _optional_, they _can_ replace it. We should only use the imperative `replace` in situations where they _definitely_ have to replace something.

Reply:
It's usually a code smell to have `runBlocking {}` in production code. I put it here because it's the most straightforward way to create a `CoroutineScope` for tests and/or small snippet and didn't want to make any assumption about the context where the client will be called (Android vs Spring vs Ktor vs ... all have various ways to create their `CoroutineScopes`). 
In a real life app, there's 99% chances that the reader will have to replace this.

Reply:
Cool thanks for the context! I'll revert to "Replace"